tasks/install.yml

```diff
@@ -97,9 +101,9 @@
   become: yes
   when: not redis_is_installed or redis_install_new_version
 
-- name: REDIS | install redis gem
+- name: REDIS | install Redis Ruby Gem
```

Comment:
According to the changelog, shouldn't we check if Redis version is [3,4]?

Reply:
I think that this conditional should be added when role supports Redis 5.x.x. 

What should do if you try to install Redis 5.x?

Reply:
Yep, you're right, for some reason I mistook #62 for a PR and thought it was already done and merged. Nothing to do here then